elastic/apm-server
# Disable sourcemap upload endpoint when data streams enabled

## Motivation/summary

If we don't do this, users will see a `202  - Accepted` response, but apm-server will fail to index sourcemaps due to lack of permissions. 

## Checklist


## How to test these changes

Run apm-server with data streams enabled and send a POST request to `http://localhost:8200/assets/v1/sourcemaps`.
The request should fail.
